iceoryx_hoofs/utility/include/iox/detail/into.inl

```diff
@@ -23,22 +23,25 @@
 namespace iox
 {
 template <typename SourceType, typename DestinationType>
-inline constexpr DestinationType from(const SourceType value)
```

Comment:
Please update the copyright header.

Reply:
@elBoberido I think you should double check the copyright headers in each file that you not just touched with a search and replace.

Reply:
@mossmaurice fixed it where appropriate but  seriously, we have to get rid of this self inflicted pain. It is not required to have a date at all, much less to have the date of the last change.